db/migrate/028_change_exit_status_to_unsigned_tinyint.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeExitStatusToUnsignedTinyint < ActiveRecord::Migration
+  def up
+    change_column :executions, :exit_status, 'tinyint(4) unsigned'
```

Comment:
AR migration should be database independent. You might want to use `limit: 2` instead of `unsigned`.